indico/htdocs/js/indico/modules/registration/form/form.js

```diff
@@ -301,6 +301,10 @@ ndRegForm.directive('ndRegForm', function($rootScope, url, sortableoptions, regF
                 }
             });
 
+            element.ready(function() {
```

Comment:
just fyi, the context/selector of `ready()` is [ignored](https://api.jquery.com/ready/) - see the "All of the following syntaxes are equivalent" section.

for the sake of consistency I would use `$(document).ready(...)` as this is what we use everywhere else in the code base.

---

However, I wonder if using the event here is actually needed. How much asynchronity is happening there? Would a `_.defer()` work as well?